homeassistant/components/geo_json_events/config_flow.py

```diff
@@ -0,0 +1,84 @@
+"""Config flow to configure the GeoJSON events integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from datetime import timedelta
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_RADIUS,
+    CONF_SCAN_INTERVAL,
+    CONF_URL,
+)
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from .const import DEFAULT_RADIUS_IN_KM, DEFAULT_SCAN_INTERVAL, DOMAIN
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL): cv.string,
+        vol.Optional(CONF_LATITUDE): cv.latitude,
+        vol.Optional(CONF_LONGITUDE): cv.longitude,
+        vol.Optional(CONF_RADIUS, default=DEFAULT_RADIUS_IN_KM): vol.Coerce(float),
+        vol.Required(
+            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
+        ): cv.positive_int,
+    }
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GeoJsonEventsFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a GeoJSON events config flow."""
+
+    async def async_step_import(self, import_config: dict[str, Any]) -> FlowResult:
+        """Import a config entry from configuration.yaml."""
+        legacy_scan_interval = import_config.get(CONF_SCAN_INTERVAL, None)
+        # Convert scan interval because it now has to be in seconds.
+        if legacy_scan_interval and isinstance(legacy_scan_interval, timedelta):
+            import_config[CONF_SCAN_INTERVAL] = legacy_scan_interval.total_seconds()
+        identifier = f"{import_config[CONF_URL]}, {import_config.get(CONF_LATITUDE, self.hass.config.latitude)}, {import_config.get(CONF_LONGITUDE, self.hass.config.longitude)}"
+        await self.async_set_unique_id(identifier)
+        self._abort_if_unique_id_configured()
```

Comment:
I'm not sure that this is a valid unique-id.
Maybe you should use instead `_async_abort_entries_match`
```suggestion
        self._async_abort_entries_match({
            CONF_URL: import_config[CONF_URL],
            CONF_LATITUDE: import_config.get(CONF_LATITUDE, self.hass.config.latitude), 
            CONF_LONGITUDE: import_config.get(CONF_LONGITUDE, self.hass.config.longitude),
            })
```

Reply:
OK, I changed to `_async_abort_entries_match`.